specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/Metadata.json

```diff
@@ -0,0 +1,777 @@
+      "properties": {
+        "contentId": {
+          "description": "Id of the content item we depend on",
+          "$ref": "#/definitions/metadataContentId"
+        },
+        "kind": {
+          "description": "Type of the content item we depend on",
+          "$ref": "#/definitions/metadataKind"
+        },
+        "version": {
+          "description": "Version of the the content item we depend on.  Can be blank, * or missing to indicate any version fulfills the dependency.  If version does not match our defined numeric format then an exact match is required.",
+          "$ref": "#/definitions/metadataVersion"
+        },
+        "name": {
+          "description": "Name of the content item",
+          "type": "string"
+        },
+        "operator": {
+          "description": "Operator used for list of dependencies in criteria array.",
+          "type": "string",
+          "enum": [
+            "AND",
+            "OR"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "operator",
+            "values": [
+              {
+                "value": "AND"
+              },
+              {
+                "value": "OR"
+              }
+            ]
+          }
+        },
+        "criteria": {
+          "description": "This is the list of dependencies we must fulfill, according to the AND/OR operator",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/metadataDependencies",
+            "description": "Dependencies for the content item, what other content items it requires to work.  Can describe more complex dependencies using a recursive/nested structure. For a single dependency an id/kind/version can be supplied or operator/criteria for complex formats."
+          }
+        }
+      }
+    },
+    "metadataProperties": {
+      "description": "Metadata property bag.",
+      "required": [
+        "contentId",
+        "parentId",
+        "kind"
+      ],
+      "type": "object",
+      "properties": {
+        "contentId": {
+          "$ref": "#/definitions/metadataContentId",
+          "description": "Static ID for the content.  Used to identify dependencies and content from solutions or community.  Hard-coded/static for out of the box content and solutions. Dynamic for user-created.  This is the resource name"
+        },
+        "parentId": {
+          "$ref": "#/definitions/metadataParentId",
+          "description": "Full parent resource ID of the content item the metadata is for.  This is the full resource ID including the scope (subscription and resource group)"
+        },
+        "version": {
+          "$ref": "#/definitions/metadataVersion",
+          "description": "Version of the content.  Default and recommended format is numeric (e.g. 1, 1.0, 1.0.0, 1.0.0.0), following ARM template best practices.  Can also be any string, but then we cannot guarantee any version checks"
+        },
+        "kind": {
```

Comment:
I would recommend to rename 'kind' field as well. Kind is a top-level property in Azure. 
Look at the list of top-level properties here [https://github.com/Azure/azure-rest-api-specs/blob/119d0a9b2e51ce8b6540bf731512229e1a983069/specification/common-types/resource-management/v2/types.json#L91](https://github.com/Azure/azure-rest-api-specs/blob/119d0a9b2e51ce8b6540bf731512229e1a983069/specification/common-types/resource-management/v2/types.json#L91)

Reply:
Just rename the 'kind' property here. Rest all places should be fine.